src/client/eppo-precomputed-client.ts

```diff
@@ -327,7 +327,7 @@ export default class EppoPrecomputedClient {
 
     if (banditEvaluation == null) {
       logger.warn(`${loggerPrefix} No assigned variation. Bandit not found: ${flagKey}`);
-      return { variation: defaultValue, action: null };
+      return { variation: this.getStringAssignment(flagKey, defaultValue), action: null };
```

Comment:
Did you consider adding a test for this situation?

Reply:
For _precomputed_, I _think_ the change should be made the edge compute method, not this client.

Basically, part of the edge compute logic should be evaluating a bandit flag whether it has an active bandit allocation or not--not the responsibility of the downstream client.

For `null`, we'd do the same as `getPrecomputedAssignment()`, and do a check like:

```ts
if (assignedVariation == null) {
  logger.warn(`[Eppo SDK] No assigned variation. Bandit flag not found: ${flagKey}`);
  return defaultValue;
}
```

But otherwise would expect something in the format `{ variation: <variation to return>, action: null}` in that case